console-extensions.ts

```diff
@@ -38,16 +41,46 @@ const extensions: EncodedExtension[] = [
   } as EncodedExtension<Separator>,
 
   {
-    type: 'console.navigation/href',
+    type: 'console.navigation/resource-ns',
     properties: {
       id: 'providers',
-      insertAfter: 'importSeparator',
-      perspective: 'admin',
       section: 'virtualization',
       name: '%plugin__forklift-console-plugin~Providers for VM Import%',
-      href: '/mtv/providers',
+      model: {
+        group: 'forklift.konveyor.io',
+        kind: 'Provider',
+        version: 'v1beta1',
+      },
+      dataAttributes: {
+        'data-quickstart-id': 'qs-nav-providers',
+        'data-test-id': 'providers-nav-item',
+      },
     },
-  } as EncodedExtension<HrefNavItem>,
+  } as EncodedExtension<ResourceNSNavItem>,
+
+  {
+    type: 'console.page/resource/list',
+    properties: {
+      component: {
+        $codeRef: 'ProvidersPage',
+      },
+      model: {
+        group: 'forklift.konveyor.io',
+        kind: 'Provider',
+        version: 'v1beta1',
+      },
+    },
+  } as EncodedExtension<ResourceListPage>,
```

Comment:
This route was forgotten and should be removed. However this requires re-writing `HostsPage` component - it has a link in the bread crumb that re-directs to `/mtv/providers/vsphere`.

Reply:
@yaacov the old page is already removed so we can just patch the `HostPage`. We could also leave it for now and create a sub-tab in the details (in PR #69 )

Reply:
@rszwajko hi,

tl;dr - for now, fix the breadcrumbs

> We could also leave it for now

The scope of 2.4.0 changed, we want to have the new providers list view working before we start working on the new detail pages, so the best fix for now is to fix the bread crumbs in the vmware hosts page. That way we can wait with the details page, and address the redesign of the hosts page later.

> and create a sub-tab in the details

We will defently address the vmware hosts page in the new design for the details page (not sure it will be a sub-tab ...)

Reply:
1. Disabled the link in the HostPage breadcrumb.
2. removed unused route